worlds/witness/__init__.py

```diff
@@ -56,15 +57,16 @@ def _get_slot_data(self):
             'panelhex_to_id': self.locat.CHECK_PANELHEX_TO_ID,
             'item_id_to_door_hexes': self.items.ITEM_ID_TO_DOOR_HEX,
             'door_hexes': self.items.DOORS,
-            'symbols_not_in_the_game': self.items.SYMBOLS_NOT_IN_THE_GAME
+            'symbols_not_in_the_game': self.items.SYMBOLS_NOT_IN_THE_GAME,
+            'disabled_panels': self.player_logic.COMPLETELY_DISABLED_CHECKS,
         }
 
     def generate_early(self):
         if not (is_option_enabled(self.world, self.player, "shuffle_symbols")
                 or get_option_value(self.world, self.player, "shuffle_doors")
                 or is_option_enabled(self.world, self.player, "shuffle_lasers")):
-            raise Exception("This Witness world doesn't have any progression items. Please turn on Symbol Shuffle, Door"
-                            " Shuffle or Laser Shuffle")
+            warning("This Witness world doesn't have any progression items. Please turn on Symbol Shuffle, Door"
```

Comment:
I suggest keeping the exception if playercount > 1

Reply:
That seems reasonable actually. I'll do that